components/bunnygrabber.py

```diff
@@ -8,12 +8,21 @@ class BunnyGrabber:
     direction = 0
 
     def set_backward(self):
+        """
+            Sets motor direction to backwards.
+        """
         self.direction = -1
 
     def set_forward(self):
+        """ Sets motor direction to forwards.
```

Comment:
Since this is a single-line docstring, the end quotes should be on the same line. Try to follow https://www.python.org/dev/peps/pep-0257/

Reply:
Also no space before/after

Reply:
This is not a useful comment.